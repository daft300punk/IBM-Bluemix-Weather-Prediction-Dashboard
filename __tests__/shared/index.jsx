import React from 'react';
import Rnd from 'react-test-renderer/shallow';

const rnd = new Rnd();

afterAll(() => {
  process.env.NODE_ENV = 'test';
});

test('Snapshot match', () => {
  let App = require('shared').default;
  rnd.render((
    <App />
  ));
  expect(rnd.getRenderOutput()).toMatchSnapshot();
  jest.resetModules();
  process.env.NODE_ENV = 'development';
  App = require('shared').default;
  rnd.render((
    <App />
  ));
  expect(rnd.getRenderOutput()).toMatchSnapshot();
  process.env.NODE_ENV = 'test';
});
