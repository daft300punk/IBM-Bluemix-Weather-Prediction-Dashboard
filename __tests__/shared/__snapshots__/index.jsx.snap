// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot match 1`] = `
<div>
  <withRouter(Routes) />
</div>
`;

exports[`Snapshot match 2`] = `
<div>
  <withRouter(Routes) />
</div>
`;
